setup.py

```diff
@@ -13,7 +13,7 @@ def read_requirements():
 
 setup(
     name='gemini_agents_toolkit',
-    version='2.2.2',
+    version='2.3.0',
```

Comment:
Version updated to signalise that the new version needs a new os env `GOOGLE_PROJECT`, for example:
```
export GOOGLE_PROJECT=gemini-trading-backend
```